### PR TITLE
Set GTM_ACCOUNT_ID for ocw-www

### PR DESCRIPTION
**Pull request by mbertrand**
#### What are the relevant tickets?
Closes #1361 

#### What's this PR do?
Sets GTM account ids for ocw-www

### CODE REVIEW

**Comment by mbertrand** [comment]
@shaidar is there a file to assign .env values for CI?

File: pillar/heroku/ocw-studio.sls [hunk 1]
```diff
@@ -56,7 +53,6 @@ heroku:
     {% set rds_endpoint = salt.boto_rds.get_endpoint('ocw-studio-db-applications-{}'.format(env_data.env)) %}
     DATABASE_URL: postgres://{{ pg_creds.data.username }}:{{ pg_creds.data.password }}@{{ rds_endpoint }}/ocw_studio
     {% endif %}
-    GTM_ACCOUNT_ID: {{ env_data.GTM_ACCOUNT_ID }}
```
Comment (shaidar) [inline pillar/heroku/ocw-studio.sls:59]: I'm not clear on why this was removed from ocw-studio? I understand that the ocw-www is the main site, but don't want to analytics on the studio site as well?

Comment (mbertrand) [reply pillar/heroku/ocw-studio.sls:59]: I think these accounts are for ocwnext, not ocw-studio - is that right @pdpinch?

Comment (pdpinch) [reply pillar/heroku/ocw-studio.sls:59]: Sorry, the ones you have in this PR are specific to OCW Studio. You should have access to confirm this in https://tagmanager.google.com/#/home (ask me to add you if you don't). Anyway here's a screenshot: 

image

I'll make up some tags for OCW Next, which will include the historical UTM tags for continuity.

Comment (mbertrand) [reply pillar/heroku/ocw-studio.sls]: I put back the existing ocw-studio values, replaced ocw-www values with `?` for now

File: pillar/apps/ocw-next-qa.sls [hunk 2]
```diff
@@ -11,6 +11,7 @@ ocw-next:
   fastly_service_id: __vault__::secret-open-courseware/rc-apps/fastly-api>data>service_id
   course_base_url: https://ocwnext-rc.odl.mit.edu/courses
   ocw_studio_base_url: https://ocw-studio-rc.odl.mit.edu/
+  gtm_account_id: GTM-???????
```
Comment (pdpinch) [inline pillar/apps/ocw-next-qa.sls:14]:
```suggestion
  gtm_account_id: GTM-PJMJGF6
```

File: pillar/apps/ocw-next-production.sls [hunk 3]
```diff
@@ -11,6 +11,7 @@ ocw-next:
   fastly_service_id: __vault__::secret-open-courseware/production-apps/fastly-api>data>service_id
   course_base_url: https://ocwnext.odl.mit.edu/courses
   ocw_studio_base_url: https://ocw-studio.odl.mit.edu/
+  gtm_account_id: GTM-???????
```
Comment (pdpinch) [inline pillar/apps/ocw-next-production.sls:14]:
```suggestion
  gtm_account_id: GTM-NMQZ25T
```

**Review by pdpinch — CHANGES_REQUESTED** [review]
Ok. I dropped the two tags on here. 

Each is configured with just one, new Google Analytics 4 account. When we launch I think we'll want to update the OCW Next Prod tag to include a reference to the old OCW Google Analytics account, so we can have continuity of data.

**Comment by shaidar** [comment]
👍 @markbreedlove I believe once this is merged it will need a deployment

**Pull request merged**